app/globals.css

```diff
@@ -1,7 +1,139 @@
-/* @tailwind base; */
+@tailwind base;
 @tailwind components;
 @tailwind utilities;
 
-body{
-  background-color: antiquewhite;
-}
\ No newline at end of file
+* {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+  box-sizing: border-box;
+}
+
+.container {
```

Comment:
when we use tailwind there is no need to define our own classes anymore. Do you prefer to use normal CSS?